### PR TITLE
[2.2.0] prepare release

### PR DESCRIPTION
**Pull request by dpfaffenbauer**

### CODE REVIEW

**Comment by solverat** [comment]
## Upgrade Issue III

image

third argument in `getDataForEditmode($fieldData, $data, false);` needs to be an array (coreshop passes a boolean value - signature changed by pimcore i guess?).

This happens if  legacy serialization is enabled (which is the default).

File: CHANGELOG-2.2.x.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 # Within 2.2
 
+
+## 2.2.0-RC.2
```
Comment (jdreesen) [inline CHANGELOG-2.2.x.md:4]:
```suggestion
## 2.2.0-RC.3
```

Comment (jdreesen) [reply CHANGELOG-2.2.x.md:4]: Or is it this?
```suggestion
## 2.2.0
```

Comment (dpfaffenbauer) [reply CHANGELOG-2.2.x.md]: thanks

**Pull request merged**